src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Home } from './Home/Home';
+import { Switch, Route } from 'react-router-dom';
+
+const App = () => {
+  return(
+    <React.Fragment>
+      {/* <Header /> */}
+      <Switch>
+        <Route 
+          exact path='/' 
+          render={() => (
+            <Home />
+            )}
```

Comment:
We'll need to map out which routes we need, and decide whether we actually want the `Home` page to be the student's list of current books.

When we get to it, we'll need to have the base path redirect to/contain a link to a login/signup page of somekind.

We'll also need to add an error-handling route.